src/i18n/translations/en/translation.json

```
@@ -1 +1,14 @@
-{}
\ No newline at end of file
+{
+  "404": {
+    "h1": "404",
+    "text": "<0>The URL you entered may be broken, or the page has been removed.</0><1> </1><2>Go back to the home page</2>"
```

```suggestion
    "errorMessage": "<0>The URL you entered may be broken, or the page has been removed.</0><1> </1><2>Go back to the home page</2>"
```

`text` feels a bit too generic to me. Could we instead make it a bit more specific so that design changes to this page allow us to better target the area on the page?

---

On that note, I was under the impression that you didn't need the tags unless it was a component. Here I'd expect to see the value as:

```
The URL you entered may be broken, or the page has been removed. <2>Go back to the home page</2>
```

---

yes indeed. i used the extractor after i did this one manually. updated it. thanks for catching that! also, updated the `text` to `errorMessage` (i actually used that same name originally but changed it to be more generic 😆 )